README.adoc

```diff
@@ -0,0 +1,183 @@
+
+==== Solving bug in Mac OS to use Docker-Compose
+
+. Expose the Keycloak port in the link:./docker-compose/docker-compose-v2.1.yaml[docker-compose-v2.1]
++
+[source,yaml]
+----
+keycloakServer:
+ image: docker.io/jboss/keycloak:4.1.0.Final
+ command: "-b 0.0.0.0 -Dkeycloak.import=/ups-keycloak-config/ups-realm-sample.json"
+ volumes:
+   - ./keycloak-realm:/ups-keycloak-config:z
+ environment:
+   KEYCLOAK_USER: ${KEYCLOAK_USER}
+   KEYCLOAK_PASSWORD: ${KEYCLOAK_PASSWORD}
+ ports:
+   - 8080:8080
+----
++
+. Replace the keycloak service host to your IP machine in the link:./docker-compose/helper/exportKeycloakHost.sh[exportKeycloakHost.sh] as follows.
++
+[source,shell]
+----
+export KEYCLOAK_SERVICE_HOST=192.168.1.14 # change for you machine IP
+----
+
+== Configuration
+
+=== Container Configuration
+
+The Unified Push Server build by default creates and registers a Docker formatted container image, aerogear/unifiedpush-configurable-container.  By default this container uses an in-memory database, in-vm amq messaging, and no authentication.  While ideal for testing, for actual use the following environment variables should be set
+
+|===
+| *Environment Variable*    | *Description*
+| `KEYCLOAK_SERVICE_HOST` | URL of a KeyCloak server providing authentication.
+| `KEYCLOAK_SERVICE_PORT` | KeyCloak service port.
+| `POSTGRES_SERVICE_PORT` | Port to connect to Postgres database.
+| `POSTGRES_SERVICE_HOST` | URL of Postgres database.
+| `POSTGRES_USER` | Postgres username to use.
+| `POSTGRES_PASSWORD` | Postgres password to use.
+| `ARTEMIS_SERVICE_HOST` | Artemis AMQ service URL.
+| `ARTEMIS_SERVICE_PORT` |  Artemis AMQ service Port.
+| `ARTEMIS_USER` |  Artemis AMQ service username.
+| `ARTEMIS_PASSWORD` |  Artemis AMQ service password.
+|===
+
+===  Unified Push Server
+
+The Unified Push Server can be configured with either System Properties (passed to the Java commandline) or Environment Variables. The two options have different formats and the following list describes them using `System Property Name`/`Env Var Name`: `Purpose`.
+
+|===
+| *Environment Variable*    | *Description*
+| `CUSTOM_AEROGEAR_APNS_PUSH_HOST` | Custom host for sending Apple push notifications. Can be used for testing.
+| `CUSTOM_AEROGEAR_APNS_PUSH_PORT` | Custom port for the Apple Push Network host.
+| `CUSTOM_AEROGEAR_FCM_PUSH_HOST` | Custom host for sending Google Firebase push notifications. Can be used for testing.
+| `UPS_REALM_NAME` | Override Keycloak Realm.
+| `KEYCLOAK_SERVICE_HOST` | Override Keycloak authentication redirect.
+| `AEROGEAR_METRICS_STORAGE_DAYS` | Override the number of days the metrics are stored (default is 30 days).
+| `ARTEMIS_URL` | URL For AMQP Server.
+| `ARTEMIS_PORT` |  PORT For AMQP Server.
+| `ARTEMIS_PASSWORD` |  Password for AMQP server.
+| `ARTEMIS_USERNAME` |  Username for AMQP server.
+|===
+
```

Comment:
@danielpassos it is missing the steps over Development. 
E.g ( local env, how to test any change made in the code locally, debug and hot deploy steps)

Reply:
Let's move forward and add it in a second moment.